packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -60,4 +62,10 @@ module.exports = {
   https: protocol === 'https',
   host: host,
   overlay: false,
+  historyApiFallback: {
+    // Paths with dots should still use the history fallback.
+    // See https://github.com/facebookincubator/create-react-app/issues/387.
+    disableDotRule: true,
+  },
```

Comment:
Would you be able to create a minimal project reproducing this? I can't replicate using the standard CRA app.